Dockerfile

```diff
@@ -7,12 +7,17 @@ COPY documentation/examples/prometheus.yml  /etc/prometheus/prometheus.yml
 COPY console_libraries/                     /usr/share/prometheus/console_libraries/
 COPY consoles/                              /usr/share/prometheus/consoles/
 
-RUN ln -s /usr/share/prometheus/console_libraries /usr/share/prometheus/consoles/ /etc/prometheus/
+RUN ln -s /usr/share/prometheus/console_libraries /usr/share/prometheus/consoles/ /etc/prometheus/ && \
+    addgroup -g 1000 prometheus && \
+    adduser -s /bin/sh -S -H -u 1000 -G prometheus prometheus && \
```

Comment:
`-D` option.
I'm not totally sure but we could potentially use the `/prometheus` folder as `HOME`.

`1000` as UID and GID, doesn't make really sense to me given that they are system user/group. Something between 100 and 999 would be more logical, or just let the command choose it, given it is a almost a clean busybox base, it will probably never change between docker builds.

Also I would prefer the `addgroup` and `adduser` commands on a separate `RUN` directive. It doesn't really make sense to put it here. I would move it over `COPY` directives, given this layer is less likely to change.